main.js

```diff
@@ -141,9 +160,52 @@ function updateTimer() {
   countdown = setInterval(function() {
     newActivity.startTimer()
   }, 1000);
+  startTimer.disabled = true;
 };
 
+function logActivity(event) {
```

Comment:
I noticed the requirement to have the markComplete method in the Activity class. I am wondering if this function is doing what that method should do. Let's ask Robbie.